### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daglar-dragomiroff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@
 - Игорь Пластов, группа: РИМ-120963
 - Дмитрий Шалагин, группа: РИМ-120963
 - Агеев Николай Сергеевич, группа: РИМ-120963 
+- Noname_test
```
Comment (100Person) [inline README.md:11]: Who is "noname"?

Comment (daglar-dragomiroff) [reply README.md:11]: "Noname" is my friend.

Comment (100Person) [inline README.md:11]: 2) Who is noname2?

Comment (daglar-dragomiroff) [reply README.md:11]: 2. Noname is my friend.

Comment (100Person) [inline README.md:11]: 3) who is noname?

Comment (100Person) [reply README.md:11]: hello 3

Comment (daglar-dragomiroff) [reply README.md:11]: 3. Noname is my friend.

Comment (100Person) [inline README.md:11]: 4. who is noname?

**Review by 100Person — APPROVED** [review]
Все ок. Можно сливать.

**Pull request closed without merging**